src/CompanyService.Business/Commands/Company/CreateCompanyCommand.cs

```diff
@@ -0,0 +1,104 @@
+﻿using LT.DigitalOffice.CompanyService.Business.Commands.Company.Interfaces;
+using LT.DigitalOffice.CompanyService.Data.Interfaces;
+using LT.DigitalOffice.CompanyService.Mappers.Db.Interfaces;
+using LT.DigitalOffice.CompanyService.Models.Db;
+using LT.DigitalOffice.CompanyService.Models.Dto.Requests;
+using LT.DigitalOffice.CompanyService.Validation.Interfaces;
+using LT.DigitalOffice.Kernel.AccessValidatorEngine.Interfaces;
+using LT.DigitalOffice.Kernel.Broker;
+using LT.DigitalOffice.Kernel.Enums;
+using LT.DigitalOffice.Kernel.Exceptions.Models;
+using LT.DigitalOffice.Kernel.Extensions;
+using LT.DigitalOffice.Kernel.FluentValidationExtensions;
+using LT.DigitalOffice.Kernel.Responses;
+using LT.DigitalOffice.Models.Broker.Requests.File;
+using LT.DigitalOffice.Models.Broker.Responses.File;
+using MassTransit;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace LT.DigitalOffice.CompanyService.Business.Commands.Company
+{
+    public class CreateCompanyCommand : ICreateCompanyCommand
+    {
+        private readonly IAccessValidator _accessValidator;
+        private readonly IDbCompanyMapper _mapper;
+        private readonly IRequestClient<IAddImageRequest> _rcAddImage;
+        private readonly ILogger<CreateCompanyCommand> _logger;
+        private readonly IHttpContextAccessor _httpContextAccessor;
+        private readonly ICreateCompanyRequestValidator _validator;
+        private readonly ICompanyRepository _repository;
+
+        private Guid? AddImage(AddImageRequest image, List<string> errors)
+        {
+            Guid creatorId = _httpContextAccessor.HttpContext.GetUserId();
+            string logMessage = "Cannot add image: name - {name}, context - {context}, extension - {extension}, creatorId - {creatorId}";
+            string errorMessage = "Cannot add company logo. Please try later.";
+
+            try
+            {
+                var response = _rcAddImage.GetResponse<IOperationResult<IAddImageResponse>>(
+                                                IAddImageRequest.CreateObj(image.Name, image.Content, image.Extension, creatorId)).Result.Message;
```

Comment:
remove extra spaces

Reply:
I did this on purpose so that the method arguments are underneath it.

Reply:
you are rigth, fixed